exercises/poker/canonical-data.json

```diff
@@ -269,12 +278,48 @@
     },
     {
       "uuid": "d0927f70-5aec-43db-aed8-1cbd1b6ee9ad",
-      "description": "both hands have straight flush, tie goes to highest-ranked card",
+      "description": "both hands have a straight flush, tie goes to highest-ranked card",
```

Comment:
If you want to fix an exercise description you're going to have to add a new test that reimplements the old one. Check out this bit of the docs: https://github.com/exercism/problem-specifications/#changing-tests

Reply:
That seems to be a bit heavy for such a tiny change.

Reply:
Indeed. In that case it will need to stay as it is.

Reply:
I found this in the docs:

> Test cases are immutable, which means that once a test case has been added, it never changes. There are two exceptions:
>  - The comments field can be mutated and thus does not require adding a new test case when changing its value.
>  - **The description field can be mutated and thus does not require adding a new test case when changing its value.**

Reply:
Ahhh, I'm so glad you checked. I had misremembered the docs. I thought the exception was the `comments` field and the `scenarios` field.